mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -607,6 +612,14 @@
     <!-- Summary for tabs preference when auto closing tabs setting is set to auto close tabs after one month-->
     <string name="close_tabs_after_one_month_summary">Close after one month</string>
 
+    <!-- Studies -->
+    <!-- Title of the remove studies button -->
+    <string name="studies_remove">Remove</string>
+    <!-- Title of the active section on the studies list -->
+    <string name="studies_active">Active</string>
+    <!-- Description for studies, it indicates why Firefox use studies and links to an article for more information. The %1 will be replaced with the corresponding article URL -->
+    <string name="studies_description"><![CDATA[Firefox may install and run studies from time to time. <a href="%1$s">Learn more</a>]]></string>
```

Comment:
Seems like we try to stay away from using CDATA in our strings (see https://github.com/mozilla-l10n/android-l10n/pull/338#issuecomment-842385895 and https://github.com/mozilla-l10n/android-l10n/pull/346#issuecomment-846358934 as an example). TBH I don't remember why, perhaps @pocmo  can give some insight into this?
Also, nit in the comment: should call out "%1$s" vs right now it's %1

Reply:
We are using `CDATA` to inject HTML code into strings,.
Now sure which would be the alternative to `CDATA`? :)

These are other examples that are `CDATA` to inject HTML into strings.

https://github.com/mozilla-mobile/fenix/blob/c45703db073c8e0a40528d08eb20fe83443f282c/app/src/main/res/values/strings.xml#L500

https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/res/values/strings.xml#L1299

https://github.com/mozilla-mobile/fenix/blob/c45703db073c8e0a40528d08eb20fe83443f282c/app/src/main/res/values/strings.xml#L1307

Reply:
Aha! I see that. I've found old conversations about this, not sure if this is relevant here? https://github.com/mozilla-mobile/fenix/issues/7965
But maybe I'm mixing up a lot of stuff :) Perhaps we don't care here since there is just a link, but please let me know!

Reply:
Don't worry, I think it's OK in this case as we just have one link, that is getting formatted with HTML :)

Reply:
Phew! Thanks Arturo for educating me on this :) I think I get it now

Reply:
Thanks to you to sharing the resources and always keeping an opened eye on strings :)

Reply:
> Now sure which would be the alternative to CDATA?

Could this be two strings?
* `Firefox may install and run studies from time to time.`
* `Learn more`

Reply:
Yes, we can.

Reply:
@pocmo I opened this PR https://github.com/mozilla-mobile/fenix/pull/20840/